.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2.1
```

Comment:
You're missing the step to upload the code coverage to codecov, see for example https://github.com/adobe/aem-core-cif-components/blob/master/.circleci/config.yml#L34-L36.

Reply:
I wanted to do this in another PR and first get the `circleCI` part working, but I can also do it in this one.